elastic/kibana
# [Painless Lab] Update server to use new es-js client

Part of https://github.com/elastic/kibana/issues/73973

This PR updates the Painless Lab app to use the new ES JS client. It also updates the `handle_es_error` util to allow the consumer to provide a custom error response.

### How to test:
Verify there have been no regressions to Painless Lab.

1. Navigate to Painless Lab in Dev Tools.
2. Verify a valid Painless script produces correct result in the "Output" panel.
3. Verify an invalid Painless script shows the error details in the "Output" panel.
